src/components/JobCard.jsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+
+const StyledBox = styled.div`
+  min-height: 228px;
+  width: 90%;
+
+  @media (min-width: ${(props) => props.theme.breakpoints.tablet_small}) {
+    width: 45%;
+  }
+
+  @media (min-width: ${(props) => props.theme.breakpoints.desktop_small}) {
+    width: 30%;
+  }
```

Comment:
I might not have been clear in my previous comment, but we do not want to define the width and height of this component since we want these attributes to be calculated based on the layout the component is placed in (width) and its content size (height). Please remove lines 6 to 15. Thank you!